api/src/main/java/org/openmrs/PatientProgram.java

```diff
@@ -57,7 +63,7 @@ public PatientProgram() {
 	public PatientProgram(Integer patientProgramId) {
 		setPatientProgramId(patientProgramId);
 	}
-	
```

Comment:
Did you intentionally leave out the default constructor?

Reply:
the default constructor is there in line 58 but not part of the changes